### PR TITLE
[PRO-141] create or import a component that can convert rich text from

### PR DESCRIPTION
**Pull request by eddie-m-m**
Fixes PRO-166

### CODE REVIEW

File: src/pages/EthicalPrinciples.tsx [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
 
 export default function EthicalPrinciples() {
+  const entry = useLoaderData() as Entry
+  const document = entry.fields.body as Document
+  const title = entry.fields.title as string
+
   return (
-    <BasicPage title={'Ethical Principles'} icon={icon}>
-      <p>ethical principles</p>
+    <BasicPage title={title} icon={icon}>
```
Comment (Timmehs) [inline src/pages/EthicalPrinciples.tsx:14]: (From convo) Lets use hardcoded title

**Review by Timmehs — COMMENTED** [review]
As discussed: factor out a <ContentfulPage> component for use directly in `router`, obviating need for separate page components here.

File: src/components/ContentfulPage.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <div
+          className="d-flex justify-content-center align-items-center bg-white rounded-circle"
+          style={{ width: 80, height: 80 }}
+        >
+          <img src={icon} alt="icon" width="50%" />
+        </div>
+      </div>
+      <h2 className="mb-4 text-center">{title}</h2>
+      {documentToReactComponents(document)}
+    </div>
```
Comment (Timmehs) [inline src/components/ContentfulPage.tsx:27]: We want to use BasicPage within the ContentfulPage component:
```suggestion
    <BasicPage title={title} icon={icon}>
      {documentToReactComponents(document)}
    </BasicPage>
```

**Review by Timmehs — COMMENTED** [review]
I can see you're still working on the merge conflicts, so I'll circle back when you have that done. In the mean time, I had a comment on ContentfulPage. Looking good all-n-all!

File: src/components/ContentfulPage.tsx (hunk 2 above)
Comment (eddie-m-m) [reply src/components/ContentfulPage.tsx]: Yeah, thanks. I'm not sure why I changed that from my original code...

**Review by Timmehs — APPROVED** [review]
Clickthrough looks great! 👏

**Pull request merged**